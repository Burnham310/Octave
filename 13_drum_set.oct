main = Section(DrumSetSymbol) instrument=3;; {
    B H S H
}
