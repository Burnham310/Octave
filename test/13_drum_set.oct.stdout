[
{
  "content": 1,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 2,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 0,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 2,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "instrument": "acoustic-guitar"
}
,
{
  "content": 0,
  "duration": 0,
  "gap": 0,
  "amp": 0,
  "instrument": "dummy"
}

]